src/Microsoft.CodeQuality.Analyzers/Core/Security/MarkVerbHandlersWithValidateAntiforgeryTokenAnalyzer.cs

```diff
@@ -0,0 +1,360 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using System;
+
+namespace Microsoft.CodeQuality.Analyzers.Security
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public class MarkVerbHandlersWithValidateAntiforgeryTokenAnalyzer : DiagnosticAnalyzer
```

Comment:
Given that the analyzer is specific to APIs in System.Web namespace, it might be better to move the analyzer into [Microsoft.NetCore.Analyzers](https://github.com/dotnet/roslyn-analyzers#microsoftnetcoreanalyzers) or [Microsoft.NetFramework.Analyzers](https://github.com/dotnet/roslyn-analyzers#microsoftnetframeworkanalyzers) based on whether the APIs are netframework specific or not.

Reply:
This particular implementation is specific to the older ASP.NET MVC (as opposed to the newer ASP.NET Core MVC), so I'll move 'em to Microsoft.NetFramework.Analyzers.